mlir/tools/rocmlir-tuning-driver/rocmlir-tuning-driver.cpp

```diff
@@ -509,6 +520,12 @@ static LogicalResult runTuningLoop(ModuleOp source) {
     }
   }
 
+  // Compilation (PassManager::run()) resets the cl opts, so we have to save the
+  // values.
+  // TODO: Is there a better way to do this?
```

Comment:
I am not sure why that is. But it is okay to keep a copy in struct instead of passing 5 different parameters